whois-nrtm4/src/main/java/net/ripe/db/nrtm4/dao/SnapshotFileRepository.java

```diff
@@ -34,20 +35,16 @@ public SnapshotFileRepository(
         this.jdbcTemplate = new JdbcTemplate(dataSource);
     }
 
-    public void save(
-        final long versionId,
-        final String name,
-        final String hash
-    ) {
+    public void insert(final PublishableSnapshotFile snapshotFile) {
         final KeyHolder keyHolder = new GeneratedKeyHolder();
         jdbcTemplate.update(connection -> {
                 final String sql = "" +
                     "INSERT INTO snapshot_file (version_id, name, hash) " +
                     "VALUES (?, ?, ?)";
                 final PreparedStatement pst = connection.prepareStatement(sql, Statement.RETURN_GENERATED_KEYS);
-                pst.setLong(1, versionId);
-                pst.setString(2, name);
-                pst.setString(3, hash);
+                pst.setLong(1, snapshotFile.getVersionId());
+                pst.setString(2, snapshotFile.getFileName());
```

Comment:
directly call update(String sql, @Nullable Object... args) method, it creates PreparedStatement automatically. Keeps it simpler?

Reply:
But this function is returning void